components/bigquery-components/pyproject.toml

```diff
@@ -1,30 +1,30 @@
-[project]
+[tool.poetry]
 name = "bigquery-components"
-authors = [
-    {name = "Example User", email = "user@example.com"},
-]
-description = "BigQuery components"
+version = "0.1.0"
+authors = ["Example User <user@example.com>"]
+description = "Bigquery components for vertex pipelines"
 readme = "README.md"
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Developers",
-    "Programming Language :: Python :: 3.7",
-]
-requires-python = ">=3.7"
-dynamic = ["version"]
-dependencies = [
-    "kfp == 1.8.21",
+    "Programming Language :: Python :: 3.9",
 ]
 
-[project.optional-dependencies]
-tests = [
-    "google-cloud-bigquery == 2.30.0",
-    "pytest >= 7.3.1,<8.0.0",
-]
+[tool.poetry.dependencies]
+python = ">=3.9,<3.11"
+Jinja2 = ">=3.0.1,<4.0.0"
+google-cloud-aiplatform = "1.24.1"
+google-cloud-pipeline-components = "1.0.42"
+google-cloud-bigquery = "2.30.0"
+kfp = "==1.8.21"
+
+[tool.poetry.group.dev.dependencies]
+pytest = ">=7.3.1,<8.0.0"
+pre-commit = ">=2.14.1,<3.0.0"
```

Comment:
the dependencies should match the previous dependencies from `pyproject.toml`:

```
[tool.poetry.dependencies]
python = ">=3.9,<3.11"
kfp = "==1.8.21"

[tool.poetry.group.dev.dependencies]
google-cloud-bigquery = "2.30.0"
pytest = ">=7.3.1,<8.0.0"
```

Reply:
cec9f3e

Reply:
cec9f3e

Reply:
Both bigquery and vertex components were squashed together in this commit